tests/e2e/client/main.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+import sys
+
+import requests
+
+
+def main(url):
+    response = requests.get(url)
+
+    print(response.text)
```

Comment:
Yes, it's just the output from the test runner which is pretty compact when
all is good, and very helpful when things go wrong.

On Mon, Apr 18, 2016, 1:34 PM Bryan Mau <notifications@github.com> wrote:

> In tests/e2e/client/main.py
> <https://github.com/GoogleCloudPlatform/python-compat-runtime/pull/77#discussion_r60129216>
> :
>
> > @@ -0,0 +1,22 @@
> > +import argparse
> > +import sys
> > +
> > +import requests
> > +
> > +
> > +def main(url):
> > +    response = requests.get(url)
> > +
> > +    print(response.text)
>
> Do we want to send this to the terminal page? As we add more and more
> tests this could be additional clutter. Could we instead do an assert?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/python-compat-runtime/pull/77/files/e4badd8fc154c61bb78146cb058d57f8b681bfb8#r60129216>
>